en/02_Developer_Guides/15_Customising_the_Admin_Interface/03_SingleRecordAdmin.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Single-record Admin Sections
+summary: Create admin UI's for managing a single record with SingleRecordAdmin
+---
+
+# Single-record admin sections
+
+The [`SingleRecordAdmin`](api:SilverStripe\Admin\SingleRecordAdmin) class lets you create an admin section which presents an edit form for a single record. Unlike [`ModelAdmin`](api:SilverStripe\Admin\ModelAdmin), there's no UI mechanism in a `SingleRecordAdmin` to swap what record you're editing.
+
+The main use cases for using `SingleRecordAdmin` are for a settings section (where a single record holds all the settings needed for that section), or editing a record that is unique for each signed-in user (such as that user's profile).
+
+```php
+namespace App\Admin;
+
+use App\Model\MySettingsModel;
+use SilverStripe\Admin\SingleRecordAdmin;
+
+class MySettingsAdmin extends SingleRecordAdmin
+{
+    private static $url_segment = 'my-settings';
+
+    private static $menu_title = 'My Settings';
+
+    private static $model_class = MySettingsModel::class;
+}
+```
+
+This admin section will fetch a record of the `MySettingsModel` class using the [`get_one()`](api:SilverStripe\ORM\DataObject::get_one()) method.
+
+If you don't want the admin section to fetch your record in this way, you can set the [`only_one_record`](api:SilverStripe\Admin\SingleRecordAdmin->only_one_record) configuration property to false. In this case you must provide another way for the admin section to know which record to edit. This could be in the form of a separate action on the controller (e.g. `edit/$ID`), or by calling [`setCurrentPageID()`](api:SilverStripe\Admin\LeftAndMain::setCurrentPageID()) in the [`init()`](api:SilverStripe\Admin\LeftAndMain::init()) method of the controller.
```

Comment:
```suggestion
If you don't want the admin section to fetch your record in this way, you can set the [`only_one_record`](api:SilverStripe\Admin\SingleRecordAdmin->restrict_to_single_record) configuration property to false. In this case you must provide another way for the admin section to know which record to edit. This could be in the form of a separate action on the controller (e.g. `edit/$ID`), or by calling [`setCurrentPageID()`](api:SilverStripe\Admin\LeftAndMain::setCurrentPageID()) in the [`init()`](api:SilverStripe\Admin\LeftAndMain::init()) method of the controller.
```

Reply:
Good catch. Fixed.